Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM centos:7.5.1804
+
+COPY . /etc/hammers
+
+RUN yum install -y epel-release
+RUN yum install -y python-pip python-devel gcc mardiadb-devel mysql-devel
```

Comment:
Probably want a `yum update` in here, and also, this can/should be one line so that all the operations fit on one Docker layer (will save space):

```Dockerfile
RUN yum update -y \
  && yum install -y python-pip python-devel gcc mardiadb-devel mysql-devel \
  && yum clean all
```